backend/app/main.py

```diff
@@ -21,6 +21,9 @@
 LOG_CONFIG = Path(__file__).parent / 'logging.yaml'
 setup_logging(LOG_CONFIG)
 
-if __name__ == "__main__":
+def start():
     import uvicorn
     uvicorn.run(app)
+
+if __name__ == "__main__":
+    start()
```

Comment:
Moim zdaniem nie opyla się tego wyciągać do innej funkcji, ale ok.

Reply:
to taka uroda setup.py, mozna rozwiazac to inaczej, ale tak jest najkrócej